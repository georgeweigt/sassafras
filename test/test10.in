* Titles

data
input y
datalines
1
2
3
;

title1 "Title 1"
title2 "Title 2"
title3 "Title 3"

proc print

proc print
