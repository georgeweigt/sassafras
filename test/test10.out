                                    Title 1
                                    Title 2
                                    Title 3

                                   Obs     Y
                                     1     1
                                     2     2
                                     3     3

                                    Title 1
                                    Title 2
                                    Title 3

                                   Obs     Y
                                     1     1
                                     2     2
                                     3     3

